src/codegen/metadata/attributes.py

```diff
@@ -2300,7 +2300,7 @@
                 }
             },
             'python_object_has_factory': True,
-            'python_object_module_location': 'nidaqmx._task_modules.channels.channel',
+            'python_object_module_location': 'nidaqmx.task.channels',
```

Comment:
I assume you will be posting a scrapigen PR for this.

Reply:
@bkeryan Yep, PR already merged in https://github.com/ni/grpc-device-scrapigen/pull/227

Reply:
Thanks, I forgot